docs/edge-authorizers/overview.mdx

```diff
@@ -13,6 +13,15 @@ instances running as close to your application as possible.
 
 <img src="/aserto-edge-authorizers.png" />
 
+## Edge Authorizers and Topaz
+
+Edge Authorizers are Topaz instances. What makes them different is how they are configured. Edge Authorizers connect to the Aserto Control Plane and light up with additional functionality:
+
+* Policy changes and data are pushed down when changes occur
+* Decisions logs are published centrally where they can be viewed from the Aserto Console or accessed via the Decision Logs API.
```

Comment:
Omri likes "captured" wdyt about "Decision logs are automatically captured and aggregated by the Aserto control plane, where they can be viewed centrally, or accessed via the Decision Logs API" 

Or maybe we should explain what decision logs are? "A log of every authorization decision made is automatically captured and aggregated by the Aserto control plane, where they can be viewed centrally, or accessed via the Decision Logs API..."

Reply:
We can link to the decision log topic. Trying to reduce duplication defining things